content/tools-and-utilities/self-assignable-roles.md

```diff
@@ -0,0 +1,213 @@
++++
+title = 'Self Assignable Roles'
+weight = 2
++++
+
+## Tutorial Video
+
+{{< youtube IaTfJ4vqHhc >}}
+
+[Tutorial Video](https://www.youtube.com/watch?v=IaTfJ4vqHhc)
+
+## Roles
+
+{{% notice warning %}}
+
+Make sure that the bot has the _manage role_ permission and that the bot's role is **above** the role it is trying to assign.
+
+{{% /notice %}}
+
+{{% notice warning %}}
+
+If you want to use any of the `rolemenu` commands, you **need** to have the `MANAGE_GUILD` permission, or the Manage Server permission. This is hardcoded, meaning that command overrides will not affect it.
+
+{{% /notice %}}
+
+\
+Simply give the role command a name and then select which role you want the bot to assign to the person.&#x20;
+
+![Creation of a role command](role_command_creation.png)
+
+### Optional features
+
+{{% notice warning %}}
+
+Do **NOT** set the require role to the role you are assigning. You generally will not want to set the ignore role to the role you are assigning either _unless_ you wish to prevent the user from removing that role through the role-menu.
```

Comment:
```suggestion
Do **not** set the require role to the role you are assigning. You generally will not want to set the ignore role to the role you are assigning either _unless_ you wish to prevent the user from removing that role through the role-menu.
```